packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -334,11 +336,61 @@ class _PrivacySettings extends StatelessWidget {
         const UserPreferencesListItemDivider(),
         const _SendAnonymousDataSetting(),
         const UserPreferencesListItemDivider(),
+        const _AdvancedSettings(),
+        const UserPreferencesListItemDivider(),
       ],
     );
   }
 }
 
+class _AdvancedSettings extends StatelessWidget {
+  const _AdvancedSettings({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return GestureDetector(
```

Comment:
Hey, listTile gives onTap functionality, we can make use of that.

Reply:
ListTile only contains the text. If I'll provide ontap() function here app-settings will only open when I'll click on text area not on the setting icon. App-settings should open when I click wherever on that particular row.

Reply:
yes, listtile gives leading and trailiing property, use that instead of seperate padding. so onTap on list tile will handle tap on anywhere on whole list.

Reply:
Yes! this should be the nice solution. Thanks @Sudhanva-Nadiger let me make changes.

Reply:
I have made changes @Sudhanva-Nadiger